### PR TITLE
Gitcoin API Client

### PR DESCRIPTION
**Pull request by dilatebrave**
https://github.com/gitcoinco/web/issues/1181

- [x] Ability to receive bounties from the gitcoin API
- [x] Ability to filter by all of the filters available in the API
- [x] Support for sorting and pagination
- [x] Readme which documents usage

### CODE REVIEW

**Comment by owocki** [comment]
nice fast work :) this is on my 'to review' list

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```
[the remark this answers is not in the document]
Comment (dilatebrave) [reply LICENSE]: i switched license in the new commit

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - "node"
```
[the remark this answers is not in the document]
Comment (mbeacom) [reply .travis.yml]: Any chance we can switch out 6 for 8 here since it's the LTS version?

File: package.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
     "sdk"
   ],
   "author": "Dilate Brave",
-  "license": "MIT",
+  "license": "AGPL-3.0",
```
Comment (mbeacom) [inline package.json]: 🙌

File: .travis.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (dilatebrave) [reply .travis.yml]: updated ^___^

**Pull request merged**